cmd/sisud/cmd/dev/deploy-and-fund.go

```diff
@@ -26,8 +26,8 @@ func DeployAndFund() *cobra.Command {
 			sisuRpc, _ := cmd.Flags().GetString(flags.SisuRpc)
 			amount, _ := cmd.Flags().GetInt(flags.Amount)
 
-			expectedErc20String := fmt.Sprintf("%s,%s", ExpectedErc20Address, ExpectedErc20Address)
```

Comment:
Do not hard code the value of ExpectedErc20Address. You can either pass empty string "," or pass new value in the command. 

If "," string is used for expectedErc20String, this command will not check if the erc20 has been deployed or not.